@PTOptions/PTOptions.m

```diff
@@ -38,8 +37,8 @@
       % Scaling factor for temperature adaptation
       temperatureEta = 100;
       
-      % Maximum T - may be infinity
-      maxT = inf;
+      % Maximum T 
+      maxT = 5e4;
```

Comment:
...is there a good reason for this number?

Reply:
thats the number benny recommended and it should better be higher than too low

Reply:
We should think about implementing his idea of aimed tempering at some point... ;)